### PR TITLE
GP008 changes/updates

### PR DESCRIPTION
**Pull request by rekjuh**
Split excercise 1 to a and b, small additions all around. Added extra excercise 6.

### CODE REVIEW

File: src/GP008/03_harjoitus_3.Rmd [hunk 1]
```diff
@@ -6,7 +6,7 @@ Harjoituksessa tehdään koropleettikartta, joka kuvaa kauppojen keskittymistä
 
 ## Harjoituksen tavoite
 
-Koulutettava oppii luomaan heksagoniruudukon, laskemaan pisteet polygoneista ja käyttämään luokittelutyyliä visualisointiin. Hän tietää myös, mistä QGISin eri väripaletit löytyvät.
+Koulutettava oppii luomaan heksagoniruudukon, laskemaan pisteet polygoneista ja käyttämään luokittelutyyliä visualisointiin. Hän tietää myös, pistä QGISin eri väripaletit löytyvät.
```
Comment (jeeemil) [inline src/GP008/03_harjoitus_3.Rmd:9]: pitäis varmaan olla "mistä"

File: src/GP008/02_harjoitus_2.Rmd [hunk 2]
```diff
@@ -47,42 +48,67 @@ Avaa seuraavaksi Sijoittaminen-välilehti. Huomaat, että se näyttää hieman e
...
 ![](img/harjoitus2/img11.png)
 
+Seuraavaksi tehdään nimiö, joka ei seuraa kohteen laitoja tai muotoa, vaan menee käyttäjän määrittämällä tavalla, tässä tapauksessa kaarella.
+
 Luo aluksi uusi väliaikainen taso painamalla Luo väliaikainen luonnostaso -painiketta.
 
 ![](img/harjoitus2/img12.png)
 
-Anna tason nimeksi Merikoski, geometriatyypiksi LineString ja koordinaattijärjestelmäksi Projektin koordinaattijärjestelmä (EPSG: 3067). Tämän jälkeen luo uusi kenttä, jonka nimi on "Nimi" ja tyyppi Teksti (merkkijono). Kun olet määrittänyt uuden kentän, paina "Lisää kenttälistaan", jolloin se tulee kenttälistaan.
+Anna tason nimeksi Merikoski, geometriatyypiksi LineString ja koordinaattijärjestelmäksi Projektin koordinaattijärjestelmä (EPSG: 3067). Tämän jälkeen luo uusi kenttä, jonka nimi on "Nimi" ja tyyppi Tekstijono. Kun olet määrittänyt uuden kentän, paina "Lisää kenttälistaan", jolloin se tulee kenttälistaan.
```
Comment (jeeemil) [inline src/GP008/02_harjoitus_2.Rmd:111]: Mulla lukaa "Teksti (merkkijono)" omassa QGISissä

File: src/GP008/06_harjoitus_6.Rmd [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Harjoituksen sisältö
+Harjoituksessa tehdään oma väripaletti ja oletusfontti.
+
+## Harjoituksen tavoite
+Koulutettava oppii luomaan valmiita väripaletteja QGISiin ja käyttämään Tyylien hallintaa.
+
+## Väripalettien luominen
+
+Voit luoda oman väripaletin kohdasta ** Asetukset /-> Valinnat /-> Värit**. Voit luoda ja nimetä uusia väripaletteja painamalla  ja lisätä olemassaoleviin uusia värejä. Jos sinulla on valmiiksi määritettyjä värejä (RGB- tai HTLM-koodeilla), voit käyttää niitä värien poimimiseen tai valita ne manuaalisesti värikartalta hiirellä.
```
Comment (jeeemil) [inline src/GP008/06_harjoitus_6.Rmd:11]: Typo: HTML

**Review by jeeemil — COMMENTED** [review]
Good job! Added a few row comments for typos!

**Comment by JuhoErvasti** [comment]
I commited some fixes relating to code/hint box styling and the exercise 6 not showing up in the TOC. Otherwise looks good to me and should be OK to merge once typos are fixed.

**Pull request merged**